lighthouse-core/lib/network-request.js

```diff
@@ -341,10 +363,89 @@ module.exports = class NetworkRequest {
     // Bail if we somehow already have transfer size data.
     if (!global.isLightrider) return;
 
-    const totalFetchedSize = this.responseHeaders.find(item => item.name === 'X-TotalFetchedSize');
+    const totalFetchedSize = this.responseHeaders.find(item => item.name === HEADER_FETCHED_SIZE);
     // Bail if the header was missing.
     if (!totalFetchedSize) return;
-    this.transferSize = parseFloat(totalFetchedSize.value);
+    const floatValue = parseFloat(totalFetchedSize.value);
+    // Bail if the header cannot be parsed
```

Comment:
```suggestion
    // Bail if the header cannot be parsed.
```
to match all the others